OsmDataCollector.py

```diff
@@ -74,7 +75,8 @@ def _collect_filtered_tracks(self):
                 for seg in track.segments:
                     if seg.points[0].time is None:  # dismisses private segments
                         continue
-                    curr_track = OsmTrack(seg)
+                    curr_track = OsmTrack(seg, self.id)
```

Comment:
As I see it, we use osmCollector in one place only -  the part that generates the osm-tracks DB.  We didn't implemented it yet, but it seems likely that it would iterate over the areas one by one using a separate osmCollector for each to create the regional DB (sequential use). 
In short, I don't think we should consider including the area in the track's id because there is no parallel use and each area has it's own DB.